databricks/koalas/testing/utils.py

```diff
@@ -131,15 +131,15 @@ def tearDownClass(cls):
         # Please see databricks/koalas/conftest.py.
         pass
 
-    def assertPandasEqual(self, left, right):
+    def assertPandasEqual(self, left, right, less_precise=False):
```

Comment:
What about we just name it `check_exact`?

Reply:
I thought about that first but I felt a little weird. I'm fine with the name if you prefer.